packages/react-microlink/src/components/Card/CardWrap.js

```diff
@@ -48,6 +51,8 @@ const style = css`
   border-style: solid;
   border-color: #E1E8ED;
   overflow: hidden;
+  color: #181919;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
maybe a good moment for extract `font-family` value and colors into a constants file 🤔

Reply:
Well, those values specifically are only used once I believe. But I'll open an issue to move colors to a file as you mention, cause the empty-state/loader could benefit from it 👌